count-stats/src/main/java/uk/ac/ebi/eva/countstats/model/CountDto.java

```diff
@@ -0,0 +1,13 @@
+package uk.ac.ebi.eva.countstats.model;
+
+import lombok.Value;
+
+import java.math.BigInteger;
+
+@Value
+public class CountDto {
```

Comment:
I think this is part of the model and should not be named `Dto`

Reply:
renamed to count